### PR TITLE
Make the dev of the day type 'DTP' to confirm the DTP step

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

File: bin/dotd [hunk 1]
```diff
@@ -152,6 +152,28 @@ def should_i(question)
   end
 end
 
+def should_i_really(question, confirmation)
+  loop do
+    print "Should I really #{question}? If so, type #{underline confirmation}: "
+    response = flush_and_gets.strip
+    puts ''
+    if response == confirmation
```
Comment (ashercodeorg) [inline bin/dotd:160]: Any reason not to compare `response.downcase` and `confirmation.downcase`?

**Comment by ashercodeorg** [comment]
For manual testing, consider something like
```
return_value = should_i_really 'test', 'DTP' do
  puts 'executed block'
end
puts "RETURN VALUE: #{return value}"
```

File: bin/dotd (hunk 1 above)
Comment (balderdash) [reply bin/dotd:160]: Just to force the dotd to be very deliberate about confirming this step. I suppose it's highly unlikely that anyone would type `dtp` without intending to deploy, but having to match the case just makes you a little slower and hopefully a little more careful.

**Comment by balderdash** [comment]
Oh sorry, I didn't mention it in the description but I manually tested it in basically the way Asher suggested. The three options work, and hitting enter without typing anything loops around as expected with a `Sorry, I didn't understand that`.

**Pull request merged**